modules/assim.batch/R/pda.emulator.R

```diff
@@ -55,10 +64,10 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
     con <- NULL
   }
 
-  bety <- PEcAn.visualization::betyConnect("~/pecan/web/config.php")
+  bety <- betyConnect("~/pecan/web/config.php")
```

Comment:
Yes for now don't use the betyConnect function - I'll go back and look at it after AGU.  You should just be able to set bety to what I had above and have it work. If not, let me know and I'll fight with it.